crates/bitcoin-da/src/service.rs

```diff
@@ -984,13 +983,17 @@ impl DaService for BitcoinService {
                                         aggregate.body,
                                         aggregate.public_key,
                                         hash,
+                                        None,
                                     );
 
                                     relevant_txs.push(relevant_tx);
                                 }
                             }
-                            ParsedLightClientTransaction::Chunk(_) => {
-                                // ignore
+                            ParsedLightClientTransaction::Chunk(chunk) => {
+                                let mut relevant_tx =
+                                    BlobWithSender::new(chunk.body, vec![0], [0; 32], None);
```

Comment:
Why `vec![0], [0; 32]`?

Reply:
Sender & Hash for the chunk are non-existent